packages/sdk-core/package.json

```diff
@@ -17,7 +17,7 @@
         "README.md"
     ],
     "scripts": {
-        "build": "yarn generate-abi-files && yarn generate-web3-types && yarn generate-graphql-types && rm -rf dist && run-p build:*",
+        "build": "run-s generate:* && rm -rf dist && run-p build:*",
```

Comment:
Does sequence come from order in scripts or alphabetically?

Reply:
order in the script

Reply:
sequentially, from order, not alphabetically